lib/active_interactor/context/errors.rb

```diff
@@ -40,10 +40,18 @@ def merge_errors!(other)
         failure_errors.merge!(other.failure_errors)
       end
 
-      def resolve_errors
```

Comment:
this Module is set for deprecation and shouldn't actually be called anymore internally.  I don't think we need a fix here.